docs/tutorials/NEB/neb_tutorial.md

```diff
@@ -121,77 +120,77 @@ The NEB part of CASTEP should pick up on the imposed ionic constraint(s) and you
  
 in the `.castep` output file.
 
+*[Shouldn't this be referenced in the main documentation, as it looks like a specific feature of the NEB code?]*
 
 ### Do I need climbing NEB?
 
-The climbing NEB method modifies the force on the highest-energy image (usually close to the middle of the band) such that it tries to climb *up* the barrier. 
-This is very useful to get accurate barrier estimates, but care must be taken when more complex pathways (i.e. with multiple maxima) are found.
+The climbing NEB method modifies the force on the highest-energy image (usually close to the middle of the band) *[? What's a band?]* such that it tries to climb *up* the barrier. 
+This is very useful to get accurate barrier estimates, but care must be taken when more complex pathways (i.e. with multiple maxima) are found.  *[Don't get this!]*
 
 In our example the path is quite straightforward and it makes sense to use the climbing image method. We therefore set:
 
 `TSSEARCH_NEB_CLIMBING: TRUE`
 
-in the .param file.
+in the .param file. *[This is the default surely? Perhaps clearer in main documentation giving an example of when you *would* set False]* 
```

Comment:
`TSSEARCH_NEB_CLIMBING` is set to False by default, according to the keywords table in the main docs.

Reply:
I think defaulting to climbing mode off is sensible since it prevents some possible convergence issues. Since the image chosen to be 'climbing' is whichever happens to the highest energy at a given step, in some cases, especially during the early stages of the optimisation, the climbing image might not always stay the same. In the worst cases this can lead to a sort of oscillating behaviour and convergence will be difficult. 

It's generally safer to start with climbing mode off, get something that is converged (or almost), then to switch on climbing mode and restart. Perhaps we should state that somewhere.

See the discussions on these pages for more: 

- https://theory.cm.utexas.edu/henkelman/research/saddle/neb/
- https://theory.cm.utexas.edu/vtsttools/neb.html#climbing-image

Reply:
```suggestion
in the .param file. *[This is the default surely? Perhaps clearer in main documentation giving an example of when you *would* set False]* 

TODO: clarify what happens when you set climbing to true.  

TODO: include advice/explanation about starting with regular NEB and turning on climbing image after a few steps. 
```

Reply:
Yes, but remove comment in L140.